domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -179,6 +180,9 @@ class StateRetriever @Inject constructor() {
       "MultipleChoiceInput" -> InteractionObject.newBuilder()
         .setNonNegativeInt(inputJson.getInt(keyName))
         .build()
+      "ItemSelectionInput" -> InteractionObject.newBuilder()
+        .setNonNegativeInt(inputJson.getInt(keyName))
```

Comment:
This isn't correct--ItemSelection uses type StringList, not negative int. Please fix.

Reply:
@BenHenning  please check this implementation.

Reply:
Changed implementation according to #270.